clients/java/signalr/src/main/java/HubConnectionState.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public enum HubConnectionState {
+    CONNECTED,
```

Comment:
Will there be other ConnectionStates besides these two? Otherwise I don't see the motivation for this change.

Reply:
There should be in the future